### ISSUE TITLE
Crashes in about an hour using Wine

### ISSUE THREAD

**Issue by IrenvanderBeekPXL** [LiveSplit/LiveSplit#2250]
Hello

I am a casual speedrunner trying to run Super Mario Odyssey, and make a video about it. I use Linux, and since LiveSplit is only availible on Windows, I tend to use Wine. During my Freddi Fish speedrun, this worked perfectly, but with SMO, it crashes about an hour in (during Night Metro). Any idea why?

I am using Kubuntu 20.04 and wine-stable (probably 7.0) and LiveSplit 1.8.19

**Comment by LukeSaward1** [comment]
Wine doesn't work with all applications. Last I tried it with Wine, it didn't go well.

**Comment by IrenvanderBeekPXL** [comment]
True, but most of the times, it works fully, or it just straight up doesn't work

**Comment by LukeSaward1** [comment]
You can't really expect good results on an application that's geared toward Windows on a compatibility layer tbh. One of the downsides about Linux, really.

**Comment by IrenvanderBeekPXL** [comment]
I know. But since I am a programmer myself, I am creating a Java app that runs native on all platforms, but not as advaced as LiveSplit

**Comment by njshockey** [comment]
You might want to checkout [Livesplit One](https://one.livesplit.org/). It works in a web browser and you can save the link as a Chrome app which might make it easier to record the window with. All splits and layouts work as far as I know.

**Comment by just-ero** [comment]
Does this issue persist on the most recent stable release of LiveSplit?
